testing/collatzConjecture.js

```diff
@@ -0,0 +1,12 @@
+import { expect } from 'chai'
+import collatzConjecture from '../functions/collatzConjecture'
+
+
+describe('#collatzConjecture', function() {
+  context('takes an integer', function() {
```

Comment:
Do you need `context` here? You're not setting up a specific set of tests that rely on a state or context to run in.